Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py

```diff
@@ -0,0 +1,832 @@
+
+        if self.user_exist_group_sta(userName=userName, groupName=groupName) is True:
+            user_id = self.get_user_id_sta(userName=userName)
+            group_id = self.get_group_id_sta(groupName=groupName)
+
+            return self.http_request(
+                method='DELETE',
+                url_suffix=urljoin(urljoin(urljoin('/groups/', quote_plus(group_id)), '/members/'),
+                                   quote_plus(user_id)),
+            )
+        else:
+            raise Exception(f"Username - {userName} is not a member of the group - {groupName}.")
+
+    # Creates a log's attribute dictionary from API's response data.
+    def logs_attributes_sta(self, response):
+
+        logs_attributes = {}
+        logs_attributes['timeStamp'] = response['timeStamp']
+        logs_attributes['userName'] = response['details']['usedName']
+        logs_attributes['resultText'] = response['details']['resultText']
+        logs_attributes['credentialType'] = response['details']['credentialType']
+        logs_attributes['ip'] = response['context']['originatingAddress']
+        logs_attributes['message'] = response['details']['message']
+        logs_attributes['serial'] = response['details']['serial']
+        logs_attributes['actionText'] = response['details']['actionText']
+
+        return logs_attributes
+
+    # Filter out the required data from total items as per limit and userName argument.
+    def logs_data_filter_sta(self, total_items, userName=None, limit=None, count=1, logs_items=None):
+
+        if logs_items is None:
+            logs_items = []
+        if userName:
+            for response in total_items:
+                if 'usedName' in response['details'].keys():
+                    if response['details']['usedName'] == userName:
+                        if limit:
+                            if limit >= count:
+                                count = count + 1
+                            else:
+                                break
+                        logs_items.append(self.logs_attributes_sta(response=response))
+
+        else:
+            for response in total_items:
+                if 'usedName' in response['details'].keys():
+                    if limit:
+                        if limit >= count:
+                            count = count + 1
+                        else:
+                            break
+                    logs_items.append(self.logs_attributes_sta(response=response))
+
+        return logs_items, count
+
+    # Get user's access logs.
+    def get_access_logs_sta(self, userName=None, since=None, until=None, limit=None):
+
+        uri = '/logs'
+        query_params = {}
+        if userName:
+            self.get_user_info_sta(userName=userName)
+        if since:
+            query_params['since'] = self.validate_date_sta(datestring=since)
+        if until:
+            query_params['until'] = self.validate_date_sta(datestring=until)
```

Comment:
We usually give an option to give a range "3 days" and we handle it in the code. 
Example: https://github.com/demisto/content/blob/master/Packs/FireEyeCM/Integrations/FireEyeCM/FireEyeCM.py#L219

Reply:
Is it mandatory ?

Reply:
As discussed, this is not mandatory. Already having filter of since, until, username.